### ISSUE TITLE
UPDATE ERROR: Server Error (500 Internal Server Error)

### ISSUE THREAD

**Issue by ancillotti** [pojome/elementor#3351]
SCENARIO:
WordPress 4.9.2
Elementor v1.9.3

PROBLEM:
After updating WordPress to the latest version, when I make a change to the page content with ELEMENTOR Page Builder, pressing [UPDATE] button returns me ERROR:

Server Error (500 Internal Server Error)

and the page is not updated

thks for the support
Claudio

**Comment by ancillotti** [comment]
I increased memory several times as recommended but the problem persists. I attached a screenshot of the error
![elementor_update_server error 500 internal server error](https://user-images.githubusercontent.com/35971027/35584214-a4e142bc-05f4-11e8-84ef-36d316668e3d.png)

**Comment by norewp** [comment]
I've had this issue pop up a few times and when it did all instances have been to do with 3rd party addon modules - so my inclination is that somewhere in their code they are doing it wrong!

I've not had this happen on native Elementor Core & Pro modules so would be of great help if you can test on those modules to see if the issue persists for you.

**Comment by ancillotti** [comment]
the problem I found when I updated Wordpress and Elementor to the latest version.

Before everything worked perfectly!

Immediately after the update when I went to edit the page created earlier with Elementor, he gave me the error message and I can not change it.

**Comment by ancillotti** [comment]
I would be happy to test the native modules Elementor Core & Pro modules could you tell me how to install them?
thanks Claudio

**Comment by bainternet** [comment]
@ancillotti 
Native core and pro are what comes with Elementor and Elementor Pro.

try to disable all other elementor related plugins you have installed and see if the issue still persists. If it does please contact our support and see how they can help you.
support@elementor.com

**Comment by Alecto** [comment]
the same error

**Comment by Miguelangel777** [comment]
Same error!!!

It must be related to some new elementor’s bug

**Comment by fulanodigital** [comment]
I've just discovered that what was causing the error was the presence of a Global Widget (a form) inserted in my page.
I tested other pages and that was the only page causing the internal error. So I started removing some widgets, saving, undoing... Untill when I removed the form, I could save normally without the internal server error.
So I unlinked it from the Global Widget and I could save it.

**Comment by severin1992** [comment]
@fulanodigital could you specify what you mean with the global widget (/form)? And where did you unlink it?
I also found out that on some pages everything works and on others not.

**Comment by mspr1nt** [comment]
Same error - also issues with the global widget.
I saved a button as a global widget and whenever I tried to add that to sections or pages it bombs. ARRRGG

**Comment by severin1992** [comment]
Aren't global widgets only available in the pro version @mspr1nt ? I don't even have it. 
Any help here highly appreciated.

**Comment by fulanodigital** [comment]
I created a contact form and saved as global widget in Elementor.

Than I inserted it in my homepage.

When you click a global widget of Elementor that was inserted in a page you
will see a button "unlink"




Em qui, 9 de ago de 2018 03:05, severin1992 <notifications@github.com>
escreveu:

**Comment by christianvgraves** [comment]
I received the same error and now the edit view won't load at all. It redirects me to this page 
<img width="1280" alt="screen shot 2018-10-17 at 2 39 29 pm" src="https://user-images.githubusercontent.com/44242382/47109116-70031400-d21b-11e8-9390-28b6fe95c16e.png">
<img width="1280" alt="screen shot 2018-10-17 at 2 39 19 pm" src="https://user-images.githubusercontent.com/44242382/47109117-70031400-d21b-11e8-9df2-b6d7ad462c3d.png">

Has this happened to anyone else?

**Comment by christianvgraves** [comment]
I've been receiving the same 500 Server Error that's now prohibiting me from even opening up the edit view altogether. It redirects me to this -- 
<img width="1271" alt="screen shot 2018-10-17 at 2 48 01 pm" src="https://user-images.githubusercontent.com/44242382/47109287-db4ce600-d21b-11e8-9229-65268eadea7e.png">
<img width="818" alt="screen shot 2018-10-17 at 2 48 25 pm" src="https://user-images.githubusercontent.com/44242382/47109288-db4ce600-d21b-11e8-944a-76986154aaa7.png">

Has this happened to anyone else?

**Comment by webk22** [comment]
I got the same error. it was working 2 days before I didn't update nothing but it was not working today.. i tried to update wordpress and any other plugin but still the same .. any help

**Comment by Stfane** [comment]
Hello, for my part I had to deactivate my form plugin (Contact Form 7) and everything is back in order

**Comment by severendis** [comment]
Hello,
I also had the problem and tried everything that was suggested here.
I did not have any global widgets. I have removed all unused plugins. No success!
I compiled my page (the problem was only on one page!) And then tried to remove elements in the copy. Even if the page was empty - no success!

I then created a new page and started to fill it with my elements again (I copied the piece by piece from one to the other window). The first Elemnet no problem - it is saved. Great, I thought.

Suddenly the same problem reappeared.

If I take a step back in the revisions, then it works again. The module I insert is a copy of one I already have (a picture with an effect) nothing special so.

Now I have searched further and came to Elementor on the hint of minimum requirements. There should be at least 128MB of memory (PHP) available. I have checked immediately on the hosting server.

The memory limit for the memory limit was 128MB.
As a test, I've increased the limit to 256MB.

After saving the server settings, I just clicked Save in my still open page of the Elementor in which the 500 server messages always come back. See there - it works!

So obviously this is a memory issue on the web server.

Ser storage is now also much faster and the website itself is also faster (faster loading times).

So just increase the limit of the memory on the server and test. Maybe it helps with one or the other on.

Have a nice day and have fun!

Uwe

**Comment by hemapopu** [comment]
@severendis Thank you soo much for your comment! You saved my life with this, I've just written 256M in the wp_config file and it worked! Thank you!

I wrote this into the cofig file after <php?

define('WP_MEMORY_LIMIT', '256M');
define( 'WP_MAX_MEMORY_LIMIT', '256M' );

And now it's ok!

**Comment by smi3tana** [comment]
> @severendis Thank you soo much for your comment! You saved my life with this, I've just written 256M in the wp_config file and it worked! Thank you!
> 
> I wrote this into the cofig file after <php?
> 
> define('WP_MEMORY_LIMIT', '256M');
> define( 'WP_MAX_MEMORY_LIMIT', '256M' );
> 
> And now it's ok!

I had trouble updating a big page. This worked for me too. Thanks

**Comment by davidmt3** [comment]
This worked! 

Even in the latest elementor and Pro versions, and with the recommended server settings I had the same problem ( error 500 ) when saving changes.

Added the code in wp-congif and it worked. 

Mine was a HUGE PAGE too.

**Comment by elytromicina** [comment]
Esto es de 2018 y no saben como sufrí con Elementor al activar el plugin del formulario, simplemente non queria funcionar, agregamos el codigo y woala!!!!  como por arte de magia, MUUUUUCHAS gracias comentario de agosto 2020

**Comment by ErfanAzm** [comment]
my error is still there even after increasing the memory limit to 512

**Comment by davefindon** [comment]
I'm a little late in answering this, but I had a similar problem with a particularly large page causing 500 internal server errors whenever trying to save the page. I had upped the memory limit as much as possible and was going back and forth with my web host tech support with no solution.

Turns out the issue was the amount of page revisions wordpress was generating for the massive page. The page was so large it must have hit some kind of limit when trying to make new revisions. I installed the free version of WP Optimize and ran the "Clean All Post Revisions" optimization under the "Database" tab: 

![image](https://user-images.githubusercontent.com/17318293/146419153-85e7c3fd-8538-431d-80bc-94fe37b32485.png)

Refreshed Elementor and it worked great. The error has popped up since then due to the amount of edits that have been requested on that page, but I just run WP Optimize again and it's all good.

Hope that helps!!
